tests/demo/latching_topic_test.py

```diff
@@ -0,0 +1,22 @@
+# QoS Profile
+from rclpy.qos import DurabilityPolicy, HistoryPolicy, QoSProfile
+
+# ROS2 interfaces
+from std_msgs.msg import String
+
+# Testing
+from ros2_easy_test import ROS2TestEnvironment, with_single_node
+
+# Module under test and interfaces
+from ..example_nodes.well_behaved import LatchingNode
+
+
+@with_single_node(
+    LatchingNode,
+    watch_topics={"/hello": String},
+    qos_profiles={
+        "/hello": QoSProfile(durability=DurabilityPolicy.TRANSIENT_LOCAL, history=HistoryPolicy.KEEP_ALL)
+    },
+)
+async def test_plain_method(env: ROS2TestEnvironment) -> None:
```

Comment:
I think there is no benefit from making this async. :)

```suggestion
def test_plain_method(env: ROS2TestEnvironment) -> None:
```

Reply:
True, that was just some remnant from copy pasting functions from somewhere else, thanks for catching that.